README.md

```diff
@@ -24,42 +24,31 @@ Using rustc nightly builds is recommanded as some unstable features are
 required.
 
 - `rustup default nightly`
-- `rustup target add thumbv6m-none-eabi`
+- `rustup target add thumbv6m-none-eabi` for Nano S and Nano X builds
+- `rustup target add thumbv8m.main-none-eabi` for Nano S+ builds
 - install [Clang](http://releases.llvm.org/download.html).
 - install an [ARM GCC toolchain](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads)
```

Comment:
Maybe add a small note about the fact that we need `arm-none-eabi-newlib`?

From my previous experience, on GNU/Linux distributions one can install:
```bash
# On Ubuntu
sudo apt install clang gcc-arm-none-eabi gcc-multilib

# On Fedora
sudo dnf install clang arm-none-eabi-gcc arm-none-eabi-newlib

# On ArchLinux
sudo pacman -S clang arm-none-eabi-gcc arm-none-eabi-newlib
```